docs/coding.rst

```diff
@@ -689,14 +691,39 @@ Because Split styles are not global, we still have to import the page-specific P
     $brand-theme: 'firefox';
 
     @import '~@mozilla-protocol/core/protocol/css/includes/lib';
-    @import '~@mozilla-protocol/core/protocol/css/components/split';
-    @import '~@mozilla-protocol/core/protocol/css/components/picto';
+    @import '~@mozilla-protocol/core/protocol/css/components/billboard';
+
+
+Import CSS Bundles
+~~~~~~~~~~~~~~~~~~
+We created pre-built CSS bundles to be used for said components due to their frequency of use. This method only requires an import into the HTML page. Since it’s already a CSS bundle, we don’t need to import it in the respective CSS page.
```

Comment:
```suggestion
We created pre-built CSS bundles to be used for some components due to their frequency of use. This method only requires an import into the HTML template. Since it’s a separate CSS bundle, we don’t need to import that component in the respective page CSS.
```